src/org/traccar/Main.java

```diff
@@ -35,6 +40,18 @@ public static void main(String[] args) throws Exception {
             Context.getWebServer().start();
         }
 
+        Timer timer = new Timer();
+        timer.scheduleAtFixedRate(new TimerTask() {
+            @Override
+            public void run() {
+                try {
+                    Context.getDataManager().clearPositionsHistory();
+                } catch (SQLException error) {
+                    Log.warning(error);
+                }
+            }
+        }, CLEAN_DELAY, CLEAN_PERIOD);
```

Comment:
this function scheduleAtFixedRate expect 2 parameters.
do you want me to use zero value for the first one, or use another function?